### PR TITLE
Anoosha foodon01

### PR DESCRIPTION
**Pull request by Anoosha-Sehar**
Genome Tracker terms

### CODE REVIEW

File: src/ontology/foodon-edit.owl [hunk 1]
```diff
@@ -16121,7 +16122,7 @@ AnnotationAssertion(dc:date obo:FOODON_00003527 "2020-11-17T23:51:38Z"^^xsd:date
 AnnotationAssertion(rdfs:label obo:FOODON_00003527 "sugarloaf cabbage plant"@en)
 SubClassOf(obo:FOODON_00003527 obo:NCBITaxon_3716)
 
-# Class: obo:FOODON_00003528 (mushroom fruitbody)
+# Class: obo:FOODON_00003528 (mushroom (fruitbody))
```
Comment (ddooley) [inline src/ontology/foodon-edit.owl:16125]: I think you might not have refreshed your github version of foodon?  I took the brackets away from "mushroom (fruitbody)" so if you could make that update that would be good.

File: src/ontology/foodon-edit.owl [hunk 2]
```diff
@@ -23879,7 +23889,7 @@ SubClassOf(obo:FOODON_03307789 ObjectSomeValuesFrom(obo:RO_0001000 obo:FOODON_03
 
 SubClassOf(obo:FOODON_03307809 ObjectSomeValuesFrom(obo:RO_0001000 obo:FOODON_03411222))
 
-# Class: obo:FOODON_03307812 (fenugreek seed (whole, dried))
+# Class: obo:FOODON_03307812 (fenugreek seed (whole))
```
Comment (ddooley) [inline src/ontology/foodon-edit.owl:23892]: Similarly its now "fenugreek seed (whole, dried)" not "fenugreek seed (whole)"

**Review by ddooley — CHANGES_REQUESTED** [review]
I think you will need to refresh your version of FoodOn, and redo these changes. Maybe drop and restart the pull request.  I'm getting ID collisions on most of the new terms with existing terms I added in the last week or two.

**Comment by ddooley** [comment]
FYI, on my side the next available foodon ID is http://purl.obolibrary.org/obo/FOODON_00004267

**Comment by Anoosha-Sehar** [comment]
Oh okay, I merged the changes from Master but I’ll check why it didn’t update the branch.But thanks for letting me know, I’ll anyways drop this request and will generate a new one.

On Monday, February 28, 2022, 5:58 PM, Damion Dooley ***@***.***> wrote:




FYI, on my side the next available foodon ID is http://purl.obolibrary.org/obo/FOODON_00004267

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Pull request closed without merging**